interfaces/builtin/login_session_control.go

```diff
@@ -0,0 +1,69 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2019 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+const loginSessionControlSummary = `allows setup of login session & seat`
+
+const loginSessionControlBaseDeclarationSlots = `
+  login-session-control:
+    allow-installation:
+      slot-snap-type:
+        - core
+    deny-auto-connection: true
+`
+
+const loginSessionControlConnectedPlugAppArmor = `
+# Description: Can setup login session & seat.
```

Comment:
Can you adjust to be:
```
# Description: Can setup login session & seat. This grants privileged access to user sessions.
```

Reply:
I think you are going to be ok, but if you have no other interfaces other than this one, are there any seccomp denials?

Reply:
I'm not sure how best to answer this. Obviously, Mir needs other interfaces (mir, wayland and opengl) and replicating its full use of org.freedesktop.login1 would be non-trivial.

Reply:
@jdstrand Having thought further I don't understand the question:  How could anything dbus (or transitively logind) does as a result of using this interface cause a seccomp denial?